stable/hlf-peer/templates/deployment.yaml

```diff
@@ -70,10 +70,12 @@ spec:
           secret:
             secretName: {{ .Values.secrets.peer.tlsClientRootCert }}
         {{- end }}
-        {{- if .Values.secrets.channel }}
-        - name: channel
+        {{- if .Values.secrets.channels }}
+        {{- range .Values.secrets.channels}}
```

Comment:
@inzamam-iqbal Please add extra space here, for readability, and indent the range block.

Reply:
i.e. `{- range .Values.secrets.channels }}`